package.json

```diff
@@ -15,6 +15,7 @@
   "license": "MIT",
   "dependencies": {
     "@apollo/client": "latest",
+    "@definitelytyped/old-header-parser": "npm:@definitelytyped/header-parser@0.0.178",
```

Comment:
I named it something different to force someone to think twice about using it. But I can just use a regular dep.